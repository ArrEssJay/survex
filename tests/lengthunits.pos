( Easting, Northing, Altitude )
(    0.00,     0.00,     0.00 ) \1
(    0.00,    10.00,     0.00 ) \2
(    0.00,    13.05,     0.00 ) \3
(    0.00,    23.05,     0.00 ) \4
(    0.00,    32.19,     0.00 ) \5
