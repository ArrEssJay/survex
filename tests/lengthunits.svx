; pos=yes warn=0
; tests length units, also test begin/end saving/restoring units
*fix 1 0 0 0
*units tape metres
1 2 10.00 0 00
*begin
*units tape feet
2 3 10.00 0 00
*end
3 4 10.00 0 00
*units tape yards
4 5 10.00 0 00
