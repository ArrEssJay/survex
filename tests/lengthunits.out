
Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 5 survey stations, joined by 4 legs.
There are 0 loops.
Total length of survey legs =  105.62ft ( 105.62ft adjusted)
Total plan length of survey legs =  105.62ft
Total vertical length of survey legs =    0.00ft
Vertical range = 0.00ft (from 5 at 0.00ft to 5 at 0.00ft)
North-South range = 105.62ft (from 5 at 105.62ft to 1 at 0.00ft)
East-West range = 0.00ft (from 5 at 0.00ft to 5 at 0.00ft)
   2 1-nodes.
   3 2-nodes.
